uvicorn/config.py

```diff
@@ -65,38 +68,44 @@
 SSL_PROTOCOL_VERSION = getattr(ssl, "PROTOCOL_TLS", ssl.PROTOCOL_SSLv23)
 
 
-LOGGING_CONFIG = {
-    "version": 1,
-    "disable_existing_loggers": False,
-    "formatters": {
-        "default": {
-            "()": "uvicorn.logging.DefaultFormatter",
-            "fmt": "%(levelprefix)s %(message)s",
-            "use_colors": None,
-        },
-        "access": {
-            "()": "uvicorn.logging.AccessFormatter",
-            "fmt": '%(levelprefix)s %(client_addr)s - "%(request_line)s" %(status_code)s',  # noqa: E501
+LOGGING_CONFIG = MappingProxyType(
```

Comment:
Not sure I get the spirit of the PR correctly, but using this doesn't make the dict immutable if that's what you intended, only the 1st level :

```
[GCC 8.3.0] on linux
from uvicorn.config import LOGGING_CONFIG
from types import MappingProxyType
m = MappingProxyType(LOGGING_CONFIG)
m['formatters']['default']['use_colors']
m['formatters']['default']['use_colors'] = True
m['formatters']['default']['use_colors'] = False
m['formatters']['default']['use_colors'] = None
m['version']
Out[9]: 1
m['version'] = 2
Traceback (most recent call last):
  File "/home/lotso/PycharmProjects/uvicorn/venv/lib/python3.8/site-packages/IPython/core/interactiveshell.py", line 3417, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-10-4ba11964fcec>", line 1, in <module>
    m['version'] = 2
TypeError: 'mappingproxy' object does not support item assignment
```

Reply:
You are right. So that I just moved the config in the dedicated `.json` file , and initialize `LOGGING_CONFIG` as a string, points to the file